lib/context.js

```diff
@@ -527,7 +527,15 @@ api.createTermDefinition = ({
         {code: 'invalid type mapping', context: localCtx});
     }
 
-    if(type !== '@id' && type !== '@vocab' && type !== '@json') {
+    if((type === '@json' || type === '@none')) {
+      if(api.processingMode(activeCtx, 1.0)) {
+        throw new JsonLdError(
+          'Invalid JSON-LD syntax; an @context @type value not be ' +
```

Comment:
Not sure exactly what this should be, perhaps:
```suggestion
          'Invalid JSON-LD syntax; an @context @type value must not be ' +
```